config.tfvars

```diff
@@ -109,8 +109,8 @@ jira_replica_count = 1
 
 # Termination grace period
 # Under certain conditions, pods may be stuck in a Terminating state which forces shared-home pvc to be stuck
-# in Terminating too causing Terraform destroy error (timing out waiting for a deleted PVC). Set termination graceful period to 0
-# if you encounter such an issue
+# in Terminating too causing Terraform destroy error (timing out waiting for a deleted PVC).
+# Termination grace period is 0 by default. You can override it if for some reason you need a different value
```

Comment:
Can we explain what does the variable do to prevent described issue? setting this value to zero means the pod will be forced shut down in 0 mins?

Reply:
Setting to 0 is to instruct kubelet to forcefully delete the pod. More info https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/